docker-compose.yml

```diff
@@ -23,9 +23,9 @@ services:
     volumes:
       - ./db_volume:/var/lib/postgresql
 
-  discovery-service:
-    image: aista/discovery-service
-    container_name: discovery-service
+  discoveryservice:
+    image: aista/eureka
+    container_name: discoveryservice
```

Comment:
we dont need container names when they are the same(i know it was there before, just saying)